tmeister/static/.babelrc

```diff
@@ -0,0 +1,16 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react",
+  ],
+  "plugins": [
+    [
+      "@babel/plugin-proposal-decorators",
+      {
+        "legacy": true
+      }
+    ],
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-object-rest-spread",
+  ]
```

Comment:
`@babel/plugin-transform-runtime` should potentially be added here. Assuming we want to use it. I think we do...

Reply:
In order to use it we have to include the entire babel runtime. For this first pass I'm skipping it.